src/status_im/extensions/core.cljs

```diff
@@ -157,7 +158,8 @@
 
 (defn- parse-json [o]
   (when o
-    (js->clj (js/JSON.parse o) :keywordize-keys true)))
+    (utils/transform-keys-to-lower-case
```

Comment:
An alternative is to just manually recreate the map using lower-case keys. The upside is that you control what ends up in the resulting map (no surprises).  And no need to introduce the new function (less code).
Are we talking about a lot of keys here?

Reply:
ok I agree, let's avoid surprises and manage keys directly.